_rules/aria-required-context-role-ff89c9.md

```diff
@@ -0,0 +1,229 @@
+
+`aria-owns` trumps ownership by closest ancestor, giving the element with [explicit semantic role](#explicit-role) of `listitem` the correct [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+
+```html
+<div role="list" aria-owns="id1">
+	<div role="navigation">
+		<div id="id1" role="listitem">List item 1</div>
+	</div>
+</div>
+```
+
+#### Passed Example 6
+
+Since implicit ownership can cross shadow boundaries, the element with the [explicit semantic role](#explicit-role) of `listitem` is contained within its [required context role](https://www.w3.org/TR/wai-aria-1.1/#scope) `list`.
+
+```html
+<div id="host" role="list"></div>
+
+<script>
+	const host = document.querySelector('#host')
+	const root = host.attachShadow({ mode: 'open' })
+	root.innerHTML = '<div role="listitem">List item 1</div>'
+</script>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The `listitem` has no [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+
+```html
+<div role="listitem">List item 1</div>
+```
+
+#### Failed Example 2
+
+The `listitem` is owned by the `tabpanel`, because it is the closest ancestor, but `tabpanel` is not the correct [context](https://www.w3.org/TR/wai-aria-1.1/#scope) for `listitem`.
+
+```html
+<div role="list">
+	<div role="tabpanel">
+		<div role="listitem">List item 1</div>
+	</div>
+</div>
+```
+
+#### Failed Example 3
+
+The `listitem` is owned by the `aria-label="menu"` div, rather than the `list`.
+
+```html
+<div role="list">
+	<div aria-label="menu">
+		<div role="listitem">List item 1</div>
+	</div>
+</div>
+```
+
+#### Failed Example 4
+
+Since explicit ownership cannot cross shadow boundaries, the element with the [explicit semantic role](#explicit-role) of `listitem` does not have a [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+
+```html
+<div role="list" aria-owns="item"></div>
+
+<div id="host"></div>
+
+<script>
+	const host = document.querySelector('#host')
+	const root = host.attachShadow({ mode: 'open' })
+	root.innerHTML = '<div id="item" role="listitem">List item 1</div>'
+</script>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+There is no element with an [explicit semantic role](#explicit-role).
+
+```html
+<ul>
+	<li>List item 1</li>
+</ul>
+```
+
+#### Inapplicable Example 2
+
+The `listitem` is not [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+```html
+<div role="listitem" style="display:none;">List item 1</div>
+```
+
+#### Inapplicable Example 3
+
+The `header` does not have a [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed in WAI-ARIA 1.1.
+
+```html
+<div role="header" aria-level="1">Hello!</div>
+<p>Welcome to my homepage!</p>
+```
+
+#### Inapplicable Example 4
+
+The `listitem` has an [explicit semantic role](#explicit-role), but it is identical to the [implicit semantic role](#implicit-role), making the element inapplicable.
+
+```html
+<ul>
+	<li role="listitem">List item 1</li>
+</ul>
+```
```

Comment:
I do not understand why we want that case to be inapplicable.
What is so special in having identical explicit and implicit role that make us discard this.

I kinda guess this is related to the fact that we also don't care about element without explicit roles, and this one essentially has no explicit role. Butt hen, I do not either fully understand why we don't care about elements without explicit role…